geometric_shapes.repos

```diff
@@ -3,13 +3,7 @@ repositories:
     type: git
     url: https://github.com/ros-planning/geometric_shapes
     version: dashing-devel
-# TODO(henningkayser): use upstream package once https://github.com/ros-planning/random_numbers/pull/18 is merged
   random_numbers:
     type: git
-    url: https://github.com/PickNikRobotics/random_numbers
-    version: ros2-devel
-# TODO(henningkayser): merge dependencies below into upstream packages
-  octomap:
```

Comment:
Why do you need a private version of octomap?
Using this private version, Travis raises a lot of issues, which it doesn't do with the default octomap package.

Reply:
See comment below